network-config-analyzer/MethodSet.py

```diff
@@ -13,21 +12,41 @@ class MethodSet(CanonicalIntervalSet):
     """
     all_methods_list = ['GET', 'HEAD', 'POST', 'PUT', 'DELETE', 'CONNECT', 'OPTIONS', 'TRACE', 'PATCH']
 
-    def __init__(self, methods_dfa):
+    def __init__(self, methods_dfa=None):
```

Comment:
we should not receive as input a MinDFA object, as the goal is to avoid any DFA computation for the methods dimension.